### PR TITLE
Check if podcast elements exist before clicking/observing

### PR DESCRIPTION
**Pull request by arkon**
Addressing feedback in #7.

### CODE REVIEW

File: src/namespaces/PlaybackNamespace.js [hunk 1]
```diff
@@ -189,9 +195,8 @@ export default class PlaybackNamespace extends GMusicNamespace {
             // Make sure that this is the first of the notifications for the
             // insertion of the song information elements.
             if (!currentTrack.equals(lastTrack)) {
-              this.emit('change:track', currentTrack);
```
Comment (MarshallOfSound) [inline src/namespaces/PlaybackNamespace.js]: Why this change?

Comment (arkon) [inline src/namespaces/PlaybackNamespace.js]: Just to make it consistent with the other ones. I can revert it if you want though.

Comment (MarshallOfSound) [inline src/namespaces/PlaybackNamespace.js]: I have a personal vendetta against scope creep, reverting would be good 👍

Comment (arkon) [inline src/namespaces/PlaybackNamespace.js]: Swapped it back, except I left a blank line deletion in by accident. I can add another commit to fix that if you want, but that might be a bit overkill...?

Comment (MarshallOfSound) [inline src/namespaces/PlaybackNamespace.js]: 😢 Idm about massive amounts of commits.  I use the "squash and merge" button on github anyway so it will end up being a single commit 👍

Comment (arkon) [inline src/namespaces/PlaybackNamespace.js]: Should be good now. 😅
